sourcegraph/sourcegraph
# Remove link to learn.sourcegraph.com from NoResultsPage

The learn site was recently deprecated and taken offline. Some pages may eventually have a new home in the docs or the about site, but for now they are broken links.

This PR removes the links to `learn.sourcegraph.com` on the No Results page and replaces them with a link to the docs.

## Test plan

- Visit no results page
- Confirm the change

## App preview:

- [Web](https://sg-web-marek-remove-learn-link.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
